Runtimes/Overlay/Windows/CRT/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+
+gyb_expand(tgmath.swift.gyb tgmath.swift)
+
+add_library(swiftCRT
+  ucrt.swift
+  Platform.swift
+  POSIXError.swift
+  TiocConstants.swift)
```

Comment:
Are you using `tgmath.swift`? Does this need a `${CMAKE_CURRENT_BINARY_DIR}/tgmath.swift`?

Reply:
Yes, `tgmath.swift` is used. I believe you don't need it as `CMAKE_CURRENT_BINARY_DIR` is implicitly looked into.

Reply:
I certainly didn't implement anything like this into CMake, so at the very least it's a missing dependency edge, but I don't think the Swift-driver will automatically scoop up random sources in `CMAKE_CURRENT_BINARY_DIR` and add them to your module implicitly.